Shopify/cli
# Update functions limit

### WHY are these changes introduced?

Fixes https://github.com/Shopify/internal-cli-foundations/issues/622

The CLI has a current limit of 10 functions per type, but the server allows up to 50 functions in total.

### WHAT is this pull request doing?

Update the functions limit to 50 in total

### How to test your changes?

- Update [this value](https://github.com/Shopify/cli/blob/44525c6e8855f21a70087969bcd5fda1543c4f56/packages/app/src/cli/constants.ts#L31) to 1
- Create a function with `p shopify app generate extension`
- `p shopify app generate extension` should now hide the functions and show a warning about the limits
- `p shopify app generate extension --path ~/tests/2023-04-12 -t cart_transform` should fail

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
